eng/WpfArcadeSdk/tools/Pbt.targets

```diff
@@ -92,11 +92,20 @@
           Condition="'$(InternalMarkupCompilation)'=='true' And Exists('$(LocalMicrosoftWinFXTargets)') "/>
 
   <!-- 
-    _WindowsDesktopSdkTargetFrameworkVersionFloor is defined in Microsoft.NET.WindowDesktop.props. 
-    This needs to be defined before Microsoft.NET.Sdk.WindowsDesktop.targets is imported. 
+    _WindowsDesktopSdkTargetFrameworkVersionFloor, _UndefinedTargetFrameworkVersion and _TargetFrameworkVersionValue are 
+    defined in Microsoft.NET.WindowDesktop.props. 
+    
+    These need to be defined before Microsoft.NET.Sdk.WindowsDesktop.targets is imported. 
   -->
   <PropertyGroup Condition="'$(InternalMarkupCompilation)'=='true' And !Exists('$(LocalMicrosoftWinFXTargets)') ">
     <_WindowsDesktopSdkTargetFrameworkVersionFloor Condition="'$(_WindowsDesktopSdkTargetFrameworkVersionFloor)' == ''">3.0</_WindowsDesktopSdkTargetFrameworkVersionFloor>
+    
+    <!--  Represents an undefined TFV value. -->
+    <_UndefinedTargetFrameworkVersion>0.0</_UndefinedTargetFrameworkVersion>
```

Comment:
I don't think I understand this change. When is this file used vs the other .targets file?

Reply:
This doesn't ship - it's used only for building dotnet/wpf repo itself. Some WPF product assemblies require XAML compilation, but we do leverage the WindowsDesktop SDK in our build. Pbt.targets is part of the XAML compilation orchestration of those product assemblies. It tries to reuse as much of the actual WindowsDesktop SDK targets/logic as possible directly.

Reply:
ah, thanks! 👍